### PR TITLE
fix: Add the Package Name options as a Env Var

### PR DESCRIPTION
**Pull request by mikelear**
run the Package Gen and publish multiple times in a pipeline not the best approach but less change to the core code and only matters for third parties generating their packages

### CODE REVIEW

File: pkg/packageGenerator/javascript/generator.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func (g *Generator) GetPackageName() string {
 
 func (g *Generator) PushPackage(packageDir string) error {
 	out, err := g.Cmd.Execute(packageDir, "npm", "publish")
-	log.Logger().Infof(out)
+	log.Logger().Infof("%s", out)
```
Comment (Skisocks) [inline pkg/packageGenerator/javascript/generator.go:68]: The linter was failing due to string formatting being called but not needed. You can just remove the f.
```suggestion
	log.Logger().Info(out)
```

**Review by Skisocks — CHANGES_REQUESTED** [review]

File: pkg/cmd/root.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ func Main() *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 			err := cmd.Help()
 			if err != nil {
-				log.Logger().Errorf(err.Error())
```
Comment (Skisocks) [inline pkg/cmd/root.go:20]: Meant for you to changes these as well :)

File: pkg/commandRunner/cmdrunner.go [hunk 3]
```diff
@@ -34,11 +34,11 @@ func (c *CommandRunner) ExecuteAndLog(dir, name string, args ...string) error {
 	log.Logger().Infof("%sRunning command%s:%s %s %s", utils.Cyan, dirString, utils.Reset, name, strings.Join(args, " "))
 	out, err := c.Execute(dir, name, args...)
 	if err != nil {
-		log.Logger().Errorf(out)
+		log.Logger().Errorf("%s", out)
```
Comment (Skisocks) [inline pkg/commandRunner/cmdrunner.go:37]: And this

File: pkg/commandRunner/cmdrunner.go [hunk 4]
```diff
@@ -34,11 +34,11 @@ func (c *CommandRunner) ExecuteAndLog(dir, name string, args ...string) error {
...
 		return err
 	}
 	if out != "" {
-		log.Logger().Infof(out)
+		log.Logger().Infof("%s", out)
```
Comment (Skisocks) [inline pkg/commandRunner/cmdrunner.go:41]: And this

File: pkg/git/git.go [hunk 5]
```diff
@@ -75,7 +75,7 @@ func (c *Client) GetDefaultBranchName(dir string) (string, error) {
 
 func (c *Client) log(message string) {
 	if message != "" {
-		log.Logger().Infof(message)
+		log.Logger().Infof("%s", message)
```
Comment (Skisocks) [inline pkg/git/git.go:78]: And this

**Review by Skisocks — CHANGES_REQUESTED** [review]

File: pkg/commandRunner/cmdrunner.go (hunk 3 above)
Comment (mikelear) [reply pkg/commandRunner/cmdrunner.go:37]: ye went through them thought there was others changed the others but must of missed this one , ahhh

File: pkg/cmd/root.go [hunk 6]
```diff
@@ -17,7 +17,7 @@ func Main() *cobra.Command {
...
+				log.Logger().Errorf("%s", err.Error())
```
Comment (Skisocks) [inline pkg/cmd/root.go:20]: Still this 😂

**Review by Skisocks — CHANGES_REQUESTED** [review]

**Review by Skisocks — APPROVED** [review]

**Pull request merged**